pkgs/by-name/tf/tftui/package.nix

```diff
@@ -19,15 +19,17 @@ python3.pkgs.buildPythonApplication rec {
   };
 
   pythonRelaxDeps = [
+    "posthog"
     "textual"
   ];
 
   nativeBuildInputs = with python3.pkgs; [
```

Comment:
```suggestion
  nativeBuildInputs = with python3Packages; [
```
Still, migrate to `python3Packages` please ;)

Reply:
$BADWORD. Sorry.

Reply:
Done